doc/articles/using-uno-ui.md

```diff
@@ -344,6 +344,10 @@ The Uno.UI TextBlock supports the Text property as well as the
 
 For more information, see the [TextBlock](https://msdn.microsoft.com/en-us/library/windows/apps/windows.ui.xaml.controls.textblock.aspx) class.
 
+---
+
+To see this controls in action, visit [UnoPlatform Playground](https://playground.platform.uno/)
```

Comment:
This addition makes sense, but would be interesting at the top instead.

Reply:
Moved to the top!